package/tests/test_models/test_flowchart.py

```diff
@@ -368,6 +369,17 @@ def test_data_node_metadata(self):
         assert data_node_metadata.filepath == "/tmp/dataset.csv"
         assert data_node_metadata.run_command == "kedro run --to-outputs=dataset"
 
+    def test_get_preview_nrows(self):
+        metadata = {"kedro-viz": {"preview": 3}}
```

Comment:
I'm guessing this test needs to be updated to use `preview_args` right?

Reply:
thanks for this. i missed it. the tests are still failing but that's because of the kedro.extras and it should go once the other PR is merged.